repository_service_tuf_worker/repository.py

```diff
@@ -553,12 +555,20 @@ def _bootstrap_online_roles(self, root: Metadata[Root]):
 
     @staticmethod
     def _task_result(
-        task: TaskName, status: bool, details: Dict[str, Any]
+        task: TaskName,
+        status: bool,
```

Comment:
I think you can remove it from the `_task_result` parameters.
If you have error not None, it means the `status` is `False`.